### PR TITLE
convert v3.5.0

### PR DESCRIPTION
**Pull request by Evan8456**
- converted redcap v3.5 to reproschema

### CODE REVIEW

**Comment by satra** [comment]
@Evan8456 - just an fyi, we may have to rerun the sequence through the new converters.

File: activities/acoustic_task/acoustic_task_schema [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "prefLabel": "acoustic_task",
     "description": "Default description",
     "schemaVersion": "1.0.0-rc4",
-    "version": "3.0.0",
+    "version": null,
```
Comment (satra) [inline activities/acoustic_task/acoustic_task_schema]: this seems to set version to` null`

**Comment by yibeichan** [comment]
@Evan8456 i'll let you know once the new converter is done

**Comment by yibeichan** [comment]
@Evan8456 hi, you used the new converter, correct? then we should be able to merge this PR

**Comment by satra** [comment]
@yibei - i think we need to do the entire set of releases sequentially through the new converter.

**Comment by yibeichan** [comment]
oh yes, @Evan8456 will you do it or I do it?

**Comment by Evan8456** [comment]
Hi @yibeichan, no worries, I'll do it 😊.

**Comment by satra** [comment]
@Evan8456 - please do communicate with @ibevers about this, so that we don't end up doing duplicate work. 

and we should set up a github action that gets triggered whenever a new release is made on the source side.

**Pull request closed without merging**